Hymns/Home/HomeContainerView.swift

```diff
@@ -4,34 +4,54 @@ import Resolver
 struct HomeContainerView: View {
 
     @State var selectedTab: HomeTab = .none
+    @State var showSplash: Bool = false
 
     var body: some View {
-        NavigationView {
-            TabView(selection: $selectedTab) {
-                HomeView(viewModel: Resolver.resolve())
-                    .tabItem {HomeTab.home.getImage(selectedTab == HomeTab.home).imageScale(.large)}
-                    .tag(HomeTab.home)
-                    .hideNavigationBar()
+        Group { () -> AnyView in
 
-                BrowseView()
-                    .tabItem { HomeTab.browse.getImage(selectedTab == HomeTab.browse).imageScale(.large)}
-                    .tag(HomeTab.browse)
-                    .hideNavigationBar()
+            if showSplash {
+                return SplashScreenView()
+                    .opacity(showSplash ? 1 : 0)
+                    .onAppear {
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 2.7) {
+                            self.showSplash = false
+                        }
+                }.eraseToAnyView()
```

Comment:
I think, instead of sticking everything in here, can we just create another view before this, call it LaunchView or something that will either show the splash, or navigate straight to the HomeContainerView. Is that possible?

Reply:
I came across this: https://stackoverflow.com/a/62225392/1907538. Do you think something like that is feasible?

Reply:
another possibility: https://stackoverflow.com/a/57274613/1907538

Since we aren't using the back button in this case, maybe we can get away with it?